tests/v2/validation/rbac/snapshotrbac/snapshot_rbac_test.go

```diff
@@ -52,17 +52,32 @@ func (etcd *SnapshotRBACTestSuite) testRKE2K3SSnapshotRBAC(role string, standard
 	switch role {
 	case rbac.ClusterOwner.String(), rbac.RestrictedAdmin.String():
 		require.NoError(etcd.T(), err)
+		log.Info("Snapshot successful!")
 
 	case rbac.ClusterMember.String(), rbac.ProjectOwner.String(), rbac.ProjectMember.String():
 		require.Error(etcd.T(), err)
 		assert.Equal(etcd.T(), "Resource type [provisioning.cattle.io.cluster] is not updatable", err.Error())
+		log.Info("Snapshot failed as expected.")
+
```

Comment:
nit, can you remove the new lines that were added on a rebase, they are in few other places too. try running gofmt.

Reply:
I've run gofmt to remove the new lines.